requirements.txt

```diff
@@ -0,0 +1 @@
+paramiko==2.9.2
```

Comment:
Consider loosening this version to `~=2.9` in the off chance this is a dependency of another package in the same environment